setup.py

```diff
@@ -36,6 +36,9 @@
             'visdom',
             'torchvision',
         ],
+        'profile': [
```

Comment:
consider adding `profile` to the `make install` invocation

Reply:
Will do.